tutorials/promscale-run-queries.md

```diff
@@ -0,0 +1,279 @@
+----------------------------+-------------+----------------------------------------------------------------------------------------------------------------------
+ 2021-01-27 18:43:48.236+00 | 0.000275625 | {"job": "prometheus", "__name__": "go_gc_duration_seconds", "instance": "localhost:9090", "quantile": "0.5"}
+ 2021-01-27 18:43:48.236+00 | 0.000165632 | {"job": "prometheus", "__name__": "go_gc_duration_seconds", "instance": "localhost:9090", "quantile": "0.25"}
+ 2021-01-27 18:43:48.236+00 | 0.000320684 | {"job": "prometheus", "__name__": "go_gc_duration_seconds", "instance": "localhost:9090", "quantile": "0.75"}
+ 2021-01-27 18:43:52.389+00 |  1.9633e-05 | {"job": "node-exporter", "__name__": "go_gc_duration_seconds", "instance": "node_exporter:9100", "quantile": "0"}
+ 2021-01-27 18:43:52.389+00 |  1.9633e-05 | {"job": "node-exporter", "__name__": "go_gc_duration_seconds", "instance": "node_exporter:9100", "quantile": "1"}
+ 2021-01-27 18:43:52.389+00 |  1.9633e-05 | {"job": "node-exporter", "__name__": "go_gc_duration_seconds", "instance": "node_exporter:9100", "quantile": "0.5"}
+```
+This query returns the label set for the metric `go_gc_duration` in JSON format. It can then be read or further interacted with. 
+
+#### 4.1.4 Advanced Query: Percentiles aggregated over time and series
+
+The query below calculates the 99th percentile over both time and series (`app_id`) for the metric named `go_gc_duration_seconds`. This metric is a measurement for how long garbage collection is taking in Go applications:
+
+``` sql
+SELECT 
+    val(instance_id) as app,
+    percentile_cont(0.99) within group(order by value) p99
+FROM 
+    go_gc_duration_seconds
+WHERE 
+    value != 'NaN' AND val(quantile_id) = '1' AND instance_id > 0
+GROUP BY instance_id 
+ORDER BY p99 desc;
+```
+
+Sample Output:
+```sql
+        app         |     p99     
+--------------------+-------------
+ node_exporter:9100 | 0.002790063
+ localhost:9090     |  0.00097977
+```
+
+The query above is uniquely enabled by Promscale, as it aggregates over both time and series and returns an accurate calculation of the percentile. Using only a PromQL query, it is not possible to accurately calculate percentiles when aggregating over both time and series.
+
+The query above is just one example of the kind of analytics Promscale can help you perform on your Prometheus monitoring data.
+
+#### 4.1.5 Filtering by labels
+
+To simplify filtering by labels, we created operators corresponding to the selectors in PromQL. 
+
+Those operators are used in a `WHERE` clause of the form `labels ? (<label_key> <operator> <pattern>)`
+
+The four operators are:
+* == matches tag values that are equal to the pattern
+* !== matches tag value that are not equal to the pattern
+* ==~ matches tag values that match the pattern regex
+* !=~ matches tag values that are not equal to the pattern regex
```

Comment:
Consider putting the actual operators in inline code styling

Reply:
Good suggested, fixed